backend/apps/slack/management/commands/slack_sync_messages.py

```diff
@@ -322,79 +322,89 @@ def _create_message_from_data(
         parent_message: Message | None = None,
     ) -> Message | None:
         """Create Message instance using from_slack pattern."""
-        if message_data.get("subtype") in {
-            "channel_join",
-            "channel_leave",
-            "bot_message",
-        } or not any(
-            [
-                message_data.get("text"),
-                message_data.get("attachments"),
-                message_data.get("files"),
-                message_data.get("blocks"),
-            ]
-        ):
-            return None
-
         try:
-            if not (slack_user_id := (message_data.get("user") or message_data.get("bot_id"))):
-                return None
+            slack_user_id = message_data.get("user")
```

Comment:
Use just one try/except for member creation where just member data be different depending on user/bot case.